### ISSUE TITLE
Admin\Contacts - External User can view contacts

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1350]
#### Description:

As an User, I want a contacts page where I can see: a filterable and searchable list of contacts.

#### Acceptance Criteria:

Given that I've clicked on the contacts tile
When I'm on the contacts table
Then I can see the contacts grid 

[External Contacts grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7763-4553&t=BzpzehtiXCvsBSWE-4)
[Internal Contacts grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7854-90&t=BzpzehtiXCvsBSWE-4)

#### Development Checklist:

- [x] Use the existing grid MUI component as a base for the contacts grid (filtering should be already included)
- [x] Add edit button but it doesn't have do anything yet
- [x] Search feature--see how it's done for Operations for an example
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by andrea-williams** [comment]
@nanyangpro where would this page go in our existing (Part 1) IA structure?

**Comment by patrickisaac** [comment]
Notes:
- Brianna and Zoey to refine #1350
- Pat to work with Zoey and Nan refining/updating additional user stories
- Andrea creating ticket related to contact tile

**Comment by shon-button** [comment]
@patriciarussellCAS 

**Reachable at:** 
https://cas-bciers-frontend-dev.apps.silver.devops.gov.bc.ca/administration/contacts

**Comment by patriciarussellCAS** [comment]
Reviewed in Test. Will create a follow-up ticket for additional features, including the "Add Contact" button. Otherwise good to close.
